src/global-table.ts

```diff
@@ -103,9 +117,14 @@ export class GlobalTable extends GlobalTableBase {
         attributeName: props.partitionKey.name,
         keyType: 'HASH',
       }],
-      replicas: [{
-        region: Stack.of(scope).region,
-      }],
+      streamSpecification: {
+        streamViewType: 'KEYS_ONLY',
+      },
+      replicas: props.replicas ?? this.replicateregions.map((regions) => {
+        return {
+          region: regions.region,
+        };
+      }),
```

Comment:
If props.replicas is defined, this will always be assigned to props.replicas. So this means if you specify a list of replicas that excluded your current region, the deployment will fail.